app/views/sessions/new.html.erb

```diff
@@ -1,13 +1,26 @@
-<h1>Login Page</h1>
-
-<%= form_tag '/login', method: :post%>
-
-<%= label_tag :email %>
-<%= text_field_tag :email %>
-
-<%= label_tag :password %>
-<%= password_field_tag :password %>
-
-<%= submit_tag "Submit" %>
-
-<%= link_to "Connect through Facebook", "/auth/facebook"%>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="col-sm-9 col-md-7 col-lg-5 mx-auto">
+        <div class="card card-signin my-5">
+          <div class="card-body">
+            <h5 class="card-title text-center">Sign In</h5>
+              <%= form_tag '/login', method: :post, :class => "form-signin" %>
+              <div class="form-label-group">
+                <%= label_tag :email %>
+                <%= text_field_tag :email %>
+              </div>
+              <div class="form-label-group">
+                <%= label_tag :password %>
+                <%= password_field_tag :password %>
+              </div>
+              <button class="btn btn-lg btn-dark btn-primary btn-block text-uppercase" type="submit" id="submit">Sign in</button>
+              <hr class="my-4">
+              <!-- <button class="btn btn-lg btn-google btn-block text-uppercase" type="submit"><i class="fab fa-google mr-2"></i> Sign in with Google</button> -->
```

Comment:
In case we add google oauth for some reason, just leaving this here. It makes the button google orange.

Reply:
It would be pretty impressive if we had both Facebook and google. Maybe another extension.

Reply:
No promises, but I have a branch that has almost all google functionality build out already. Should be easy to add back in (watch it take 5 hours of trouble shooting).